rock.osdeps

```diff
@@ -235,7 +235,7 @@ dc1394:
 
 vtk:
     debian,ubuntu:
-        '14.04,14.10,15.04,15.10':
+        '14.04,14.10,15.04,15.10,16.04':
```

Comment:
Then you should probably revert the change (i.e. remove the 'default' entry and replace the 14.04 ... version list by default. And make sure that someone tried it on a clean install (i.e. a clean VM)